tmexp/utils.py

```diff
@@ -49,11 +55,20 @@ def create_logger(log_level: str, name: str) -> Logger:
     return logger
 
 
-def check_exists(file_path: str) -> None:
+def check_file_exists(file_path: str) -> None:
     if not os.path.exists(file_path):
         raise RuntimeError("File '%s' does not exists, aborting." % file_path)
```

Comment:
Not from this PR but we should use FileNotFoundError

Reply:
got it